### ISSUE TITLE
terminate called after throwing an instance of 'std::system_error'

### ISSUE THREAD

**Issue by LotusWang0723** [tensorflow/tensorflow#17216]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:  Ubuntu 16.04.1 LTS (GNU/Linux3.13.0-105-generic x86_64)
- **TensorFlow installed from (source or binary)**: binary (tf_nightly-1.head-cp36-cp36m-linux_x86_64.whl)
- **TensorFlow version (use command below)**: 1.7.0-dev20180222
- **Python version**: 3.6
- **Bazel version (if compiling from source)**: N/A
- **GCC/Compiler version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: CPU(Intel Xeon E5-2620 v3)
- **GPU model and memory**: N/A
- **Exact command to reproduce**:

```
>>> import tensorflow as tf
>>> hello = tf.constant('Hello, TensorFlow!')
>>> sess = tf.Session()
```

### Describe the problem
After I installed tensorflow which was downloaded from the main page you provided on github

[tf_nightly-1.head-cp36-cp36m-linux_x86_64.whl](http://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3.6,label=gpu-linux/lastSuccessfulBuild/artifact/pip_test/whl/tf_nightly_gpu-1.head-cp36-cp36m-linux_x86_64.whl)

I tried to  test the tensorflow and when I run `sess = tf.Session()`

I got the error:

```
terminate called after throwing an instance of 'std::system_error'
  what():  Resource temporarily unavailable
Aborted
```

I use anaconda create a new python 3.6 enviroment for tensorflow. After I downloaded the .whl package I use 

`pip install tf_nightly-1.head-cp36-cp36m-linux_x86_64.whl`

to install the tensorflow

I've seen a similar problem and in that case his system is CentOS. So is anyone having a similar experience with this problem?

**Comment by angersson** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by theonegis** [comment]
@LotusWang0723 Do you solve this problem? I have the same problem.

**Comment by theonegis** [comment]
I just run "tf.test.is_gpu_available()" and get the same error, can anyone help me?

**Comment by LotusWang0723** [comment]
I upgraded the CUDA and cudnn, then the error was gone.


王落桐

Lotus WANG (Ms.)

博士生

北京大学 工学院
生物医学工程系
Ph.D Candidate
Department of Biomedical Engineering

College of Engineering, Peking University


原始邮件
发件人:Victor Tannotifications@github.com
收件人:tensorflow/tensorflowtensorflow@noreply.github.com
抄送:LotusWangwangluotong@126.com; Mentionmention@noreply.github.com
发送时间:2019年3月20日(周三) 09:26
主题:Re: [tensorflow/tensorflow] terminate called after throwing aninstance of 'std::system_error' (#17216)


I just run "tf.test.is_gpu_available()" and get the same error, can anyone help me?

**Comment by theonegis** [comment]
@LotusWang0723 I have no root privilege, so I use conda to install cudatoolkit and cudnn. The versions matches with the official built binary. But I still got this error.

**Comment by theonegis** [comment]
By the way, in the same context (cudatookit=10.0, cudnn=7.4), PyTorch library can run without any issues, but TensorFlow just throws this error.

**Comment by theonegis** [comment]
I solved this problem by increasing the max user processes.

**Comment by Zen3515** [comment]
> I solved this problem by increasing the max user processes.

@theonegis Can you elaborate on how to do that?

**Comment by oksyassine** [comment]
> I solved this problem by increasing the max user processes.

@theonegis Could you please let me know what is the max user processes did you choose to fix the problem please ?

**Comment by ckolluru** [comment]
@oksyassine Run ulimit, check the max user processes allowed (-u)

Run ulimit -u 4096 (if 4096 is the max user processes allowed).

This shouldn't need root access.
